src/Microsoft.Health.Dicom.Client/Models/Partition.cs

```diff
@@ -9,17 +9,16 @@
 
 namespace Microsoft.Health.Dicom.Client.Models;
 
-[Serializable]
 public class Partition
 {
     public const string DefaultName = "Microsoft.Default";
 
     public const int DefaultKey = 1;
 
-    [JsonPropertyName("PartitionKey")]
+    [JsonPropertyName("partitionKey")] // these explicit names are here for the REST schema
     public int Key { get; }
 
-    [JsonPropertyName("PartitionName")]
+    [JsonPropertyName("partitionName")] // these explicit names are here for the REST schema
```

Comment:
There must be a way for us to incorporate camelCase names as a setting for Swagger gen (like we can for JSON.NET and System.Text.Json), but I won't block on that here